tests/SilverStripeRemoteRepositoryTest.php

```diff
@@ -3,75 +3,161 @@
 namespace SilverStripe\ApiDocs\Test;
 
 use PHPUnit\Framework\TestCase;
+use SilverStripe\ApiDocs\Data\Config;
 use SilverStripe\ApiDocs\RemoteRepository\SilverStripeRemoteRepository;
 
 class SilverStripeRemoteRepositoryTest extends TestCase
 {
-    public function testGetRepoUrl()
-    {
-        $remoteRepo = new SilverStripeRemoteRepository(
-            '',
-            '/mnt/Dev/@code-lts/@doctum-fork/api.silverstripe.org/data/packages/'
-        );
-
-        $url = $remoteRepo->getFileUrl(
-            '4',
-            'silverstripe/admin/code/AdminRootController.php',
-            0
-        );
+    private static ?string $fileContent = null;
 
-        $this->assertSame(
-            'https://github.com/silverstripe/silverstripe-admin/blob/1/code/AdminRootController.php#L0',
-            $url
-        );
-
-        $url = $remoteRepo->getFileUrl(
-            '4',
-            'silverstripe/graphql/src/Extensions/DevBuildExtension.php',
-            0
-        );
-
-        $this->assertSame(
-            'https://github.com/silverstripe/'
-            . 'silverstripe-graphql/blob/4/src/Extensions/DevBuildExtension.php#L0',
-            $url
-        );
-
-        $url = $remoteRepo->getFileUrl(
-            '5',
-            'silverstripe/graphql/src/Extensions/DevBuildExtension.php',
-            0
-        );
+    public static function setUpBeforeClass(): void
+    {
+        $config = Config::getConfig();
+        $filePath = Config::configPath($config['paths']['versionmap']);
+        if (file_exists($filePath)) {
+            self::$fileContent = file_get_contents($filePath);
+            unlink($filePath);
+        }
+        file_put_contents($filePath, json_encode([
```

Comment:
We need to make sure there's some mapping available to test against. But if you're running tests locally you may already have a map there so we make sure to keep that and restore it after the test.